gridss.wdl

```diff
@@ -61,7 +63,7 @@ task GRIDSS {
 
     runtime {
         cpu: threads
-        memory: "32G"
+        memory: "~{jvmHeapSizeGb}G"
```

Comment:
It is good to do a `+1` here to account for the overhead of the jvm and the container.